content/clause-interleaving.md

```diff
@@ -103,14 +103,13 @@ It provides the best of all worlds:
 
 ### Specification
 We amend the syntax of def parameter clauses as follows:
-
 ~~~
 DefDcl                 ::=  DefSig ‘:’ Type
 DefDef                 ::=  DefSig [‘:’ Type] ‘=’ Expr
-DefSig                 ::=  id [DefParamClauses] [DefImplicitClause]
-DefParamClauses        ::=  DefParamClauseChunk {DefParamClauseChunk}
-DefParamClauseChunk    ::=  [DefTypeParamClause] TermOrUsingParamClause {TermOrUsingParamClause}
-TermOrUsingParamClause ::=  DefTermParamClause
+DefSig                 ::=  id [DefParamClauses] [DefImplicitClause]    -- and two DefTypeParamClause cannot be adjacent
+DefParamClauses        ::=  DefParamClause { DefParamClause }
```

Comment:
```suggestion
DefSig                 ::=  id [DefParamClauses] [DefImplicitClause]
DefParamClauses        ::=  DefParamClause { DefParamClause }    -- and two DefTypeParamClause cannot be adjacent
```
Suggestion from @chrisandrews-ms
(I couldn't put it in the relevant conversation because of the way `suggestion` works)